.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+  
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
+        operating-system: [ubuntu-18.04]
+        env: ['EXECUTOR= DEPENDENCIES=--prefer-lowest', 'EXECUTOR=coroutine DEPENDENCIES=--prefer-lowest', 'EXECUTOR=', 'EXECUTOR=coroutine']
+        include:
+          - os: ubuntu-18.04
+            php: 7.1
+            env: EXECUTOR=coroutine
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install PHP
+      uses: shivammathur/setup-php@1.5.2
+      with:
+        php-version: ${{ matrix.php  }}
+        extension-csv: dom, mbstring
+        coverage: xdebug
+    - name: Get Composer Cache Directory
+      id: composer-cache
+      run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+    - name: Cache dependencies
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+        restore-keys: ${{ runner.os }}-composer-
+    - name: Install dependencies
+      run: |
```

Comment:
Can this be separated into multiple commands? Kind of weird how it's hidden

![image](https://user-images.githubusercontent.com/327717/69640938-8bb42d00-105f-11ea-9a1d-dd53b21ac417.png)

![image](https://user-images.githubusercontent.com/327717/69640927-8525b580-105f-11ea-9728-8e93f056ede8.png)

Reply:
Do you want it to be in different steps?

Reply:
Perhaps? I mean something like this

![image](https://user-images.githubusercontent.com/327717/69912992-f5528380-1431-11ea-9461-cc780b2e6384.png)

So commands are visible at first sight. Currently, significant `composer update` is hidden under `Run export $ENV` which is not significant for contributors.

Reply:
Export needed because env might multiple env variables and i didn't find beatiful solving for export